src/pages/dashboard.js

```diff
@@ -245,17 +247,41 @@ const UserContacts = () => {
 }
 
 const DashboardSideBar = () => {
-  const dropdownSkills = sampleUserData.data.getUserById.skills.map(
-    (skill) => ({
-      name: skill.name,
-      label: skill.name,
-    })
-  )
+  const { loading, error, data } = useQuery(GET_ALL_SKILLS, {
```

Comment:
A lot of the content on this page depends on data from the database (skills, education, work experience, user info). This query should be moved inside `getServerSideProps` so that the page won't be rendered until the data is loaded. This is prevents the side bar from "popping in"

https://nextjs.org/docs/basic-features/data-fetching#getserversideprops-server-side-rendering

Reply:
I just fixed the dashboard so that it uses getServerSideProps!